ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -403,10 +413,11 @@
     "CaseFieldID": "respondent1DQFileDirectionsQuestionnaire",
     "PageDisplayOrder": 6,
     "PageFieldDisplayOrder": 1,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "OPTIONAL",
```

Comment:
change this to mandatory 
if its not possible to make the checkbox alone in the complex type optional then it will be a ccd constraint.